### PR TITLE
[Accuracy diff No. 110, 111] Fix accuracy diff for paddle.nn.functional.binary_cross_entropy, paddle.nn.functional.binary_cross_entropy_with_logits API

### PR DESCRIPTION
**Pull request by NKNaN**
目前 paddle.nn.functional.binary_cross_entropy, paddle.nn.functional.binary_cross_entropy_with_logits 对应的反向算子只计算 x_grad 不计算 label_grad，而 torch 计算 x_grad 和 target_grad (label_grad)：
image
image

第一次回测结果：
image

paddle.nn.functional.binary_cross_entropy_with_logits 能全部通过
paddle.nn.functional.binary_cross_entropy 有两个 case 未通过：
```bash
paddle.nn.functional.binary_cross_entropy(Tensor([195840, 80],"float16"), Tensor([195840, 80],"float16"), reduction="none", )
paddle.nn.functional.binary_cross_entropy(Tensor([87040, 80],"float16"), Tensor([87040, 80],"float16"), reduction="none", )
```

对这两个 case 二次回测结果：
image

### CODE REVIEW

**Comment by NKNaN** [comment]
> 如果修改可以全部 pass 的话，请将所有会报 [not compare] 错误的配置，从：tester/api_config/5_accuracy/accuracy_7.txt 移动至 tester/api_config/5_accuracy/accuracy_gpu_error_grads_diff.txt，便于后续复测以及修复～

好的

**Comment by cangtianhuang** [comment]
@NKNaN 以及 tester/api_config/5_accuracy/accuracy_gpu_error.txt 中的所有 [not compare] 出错配置，也复制一份到 tester/api_config/5_accuracy/accuracy_gpu_error_grads_diff.txt 中，保留原有的便于复测 ，辛苦同学

**Comment by NKNaN** [comment]
好的，` tester/api_config/5_accuracy/accuracy_gpu_error.txt 中的所有 [not compare] 出错配置 ` 是 15857 个，在上面回测通过；` tester/api_config/5_accuracy/accuracy_7.txt ` 是 602 个，回测全部通过：
image

**Comment by NKNaN** [comment]
感谢！
accuracy_7.txt 的 602 个在修复前可以正常pass
image

已保留 accuracy_gpu_error.txt 中原有的的配置

**Comment by cangtianhuang** [comment]
> 我 revert 一下

好滴，看上去误删了很多，这个应该是 git 索引的问题，文件偏大就会有这个问题😂

**Comment by NKNaN** [comment]
> > 我 revert 一下
> 
> 好滴，看上去误删了很多，这个应该是 git 索引的问题，文件偏大就会有这个问题😂

重新提了一个 pr

**Comment by cangtianhuang** [comment]
this pr can be closed safely
